src/ReflectionFileNamespace.php

```diff
@@ -83,6 +83,9 @@ class ReflectionFileNamespace
      */
     public function __construct($fileName, $namespaceName, Namespace_ $namespaceNode = null)
     {
+        if (!is_string($fileName)) {
```

Comment:
While there are 3 places in the code where `Go\ParserReflection\ReflectionFileNamespace` is instantiated, and two of them are called with filename coming from `Object(Go\ParserReflection\ValueResolver\NodeExpressionResolver)->context->getFileName()` where `context` isn't a strongly typed property, I believe the analysis from `Go\ParserReflection\ReflectionFile`'s constructor still holds: It's a good idea to do type checking where appropriate, and I think this can stay, unless @lisachenko wants me to yank it.